tests/app/views/contexts/summary/test_question.py

```diff
@@ -147,13 +225,17 @@ def test_create_question_with_answer_label_when_empty_title(self):
 
     def test_concatenate_textfield_answers(self):
         answer_separators = {"Newline": "<br>", "Space": " "}
+        self.schema = ADDRESS_QUESTIONNAIRE_SCHEMA
 
         for concatenation_type, concatenation_character in answer_separators.items():
             with self.subTest(
                 concatenation_type=concatenation_type,
                 concatenation_character=concatenation_character,
             ):
                 # Given
+                self.answer_store.add_or_update(
+                    Answer(answer_id="building", value="Government Buildings")
```

Comment:
Would like to see this distinctly different to the default

Reply:
Changed, so it has different value than default now. Thanks!